apps/testingaccessibility/src/utils/algolia/sanity-algolia-client.ts

```diff
@@ -21,9 +21,7 @@ const algoliaWriteKeySchema = z
   })
 
 const getAlgoliaApiWriteKeyFromEnvironment = () => {
-  // TODO: make sure this doesn't break production /learn page
-  // return algoliaWriteKeySchema.parse(process.env.ALGOLIA_API_WRITE_KEY)
-  return process.env.ALGOLIA_API_WRITE_KEY
+  return algoliaWriteKeySchema.parse(process.env.ALGOLIA_API_WRITE_KEY)
```

Comment:
@vojtaholik had to comment this out because it was breaking the learn page. It can be restored now 🎉